P3826/P3826.md

```diff
@@ -1348,15 +1347,15 @@ follows:]{.ednote}
 >     friend bool operator==(const task_scheduler& lhs, const Sch& rhs) noexcept;
 >
 >   private:
->     shared_ptr<@[`void`]{.rm}[`parallel_scheduler_backend`]{.add}@> sch_; // exposition only
+>     shared_ptr<@[`void`]{.rm}[`parallel_scheduler_backend`]{.add}@> @_`sch_`_@; // exposition only
```

Comment:
`sch_` in the current working draft is italic. So, I fixed that in the wording accordingly